src/stdlib/src/print.cpp

```diff
@@ -0,0 +1,62 @@
+#include <stdint.h>
+#include <stdio.h>
+
+// fmt is an open-source formatting library providing a fast and safe alternative to C stdio and C++ iostreams.
+// https://github.com/fmtlib/fmt
+#define FMT_HEADER_ONLY
+#include "fmt/format.h"
+
+namespace perlang
+{
+    void print(const char* str)
+    {
+        // For plain strings, there's no need to use the overhead which `printf` induces. `puts` can potentially be a
+        // tiny bit faster.
+        puts(str);
+    }
+
+    void print(bool b)
+    {
+        if (b) {
+            puts("true");
+        }
+        else {
+            puts("false");
+        }
+    }
+
+    void print(char c)
+    {
+        printf("%c\n", c);
+    }
+
+    void print(int32_t i)
+    {
+        printf("%d\n", i);
+    }
+
+    void print(uint32_t u)
+    {
+        printf("%u\n", u);
+    }
+
+    void print(int64_t i)
+    {
+        printf("%ld\n", i);
+    }
+
+    void print(uint64_t u)
+    {
+        printf("%lu\n", u);
+    }
+
+    void print(float f)
+    {
+        fmt::println("{}", f);
+    }
+
+    void print(double d)
+    {
+        fmt::println("{}", d);
+    }
+}
```

Comment:
:point_up: This is one of the more important parts in this PR. It features the implementation of the `perlang::print()` methods, with various overloads for the different primitive types we support.